src/argilla/client/feedback/dataset/remote/dataset.py

```diff
@@ -550,6 +553,12 @@ def metadata_properties(self) -> List["AllowedRemoteMetadataPropertyTypes"]:
         """
         return ArgillaMetadataPropertiesMixin.list(client=self._client, dataset_id=self.id)
 
+    @property
+    def vector_settings(self) -> List[RemoteVectorSettings]:
+        """Retrieves the `vector_settings` of the current dataset from Argilla"""
+        response_model = datasets_api_v1.list_vector_settings(client=self._client, id=self.id).parsed
+        return [RemoteVectorSettings.from_orm(vector_settings) for vector_settings in response_model.items]
```

Comment:
Maybe use `response` instead?

```suggestion
        response = datasets_api_v1.list_vector_settings(client=self._client, id=self.id)
        
        return [RemoteVectorSettings.from_orm(vector_settings) for vector_settings in response.parsed.items]
```